### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by arunkarthicknm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,28 @@
 <h1 align="center">NoMoreDomains</h1>
-Need to stop buying domain names? No More Domains blocks all the best-known domain registrars and domain name auction sites to help you stop buying so many domains!
 
-⭐Leave a Star!⭐
+
+### Need to stop buying domain names?
```
Comment (immattdavison) [inline README.md:4]: This does not reflect heading hierarchy best parctices.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
 
 ## The Future
 
-NoMoreDomains is only in early development, we plan to add a redirect to a branded page eventually and create a website for NoMoreDomains. As NoMoreDomains is open source we rely solely on contributions to build this chrome extension. If you wish to make a contribution please visit the [issues](https://github.com/immattdavison/NoMoreDomains/issues) and find issues that need assistance or create one of your own!
+* NoMoreDomains is only in early development, we plan to add a redirect to a branded page eventually and create a website for NoMoreDomains. As NoMoreDomains is open source we rely solely on contributions to build this chrome extension. 
+* If you wish to make a contribution please visit the [issues](https://github.com/immattdavison/NoMoreDomains/issues) and find issues that need assistance or create one of your own!
+
+* When making contributions please consider and follow our [Contributing Guidelines](https://github.com/immattdavison/NoMoreDomains/blob/master/contributing.md).
+-------
+* This project is **Hacktoberfest 2022** accepted !
```
Comment (immattdavison) [inline README.md:22]: This is unnecessary

File: README.md [hunk 3]
```diff
@@ -1,14 +1,28 @@
...
+
+  Happy hacking :D
```
Comment (immattdavison) [inline README.md:24]: I don't think this line is necessary

**Review by immattdavison — CHANGES_REQUESTED** [review]

**Comment by arunkarthicknm** [comment]
Sure will change it 👍🏻
But that Hactoberfest accepted tagline is mentioned in the issue itself. Should I remove it?

**Comment by immattdavison** [comment]
> Sure will change it 👍🏻
> 
> But that Hactoberfest accepted tagline is mentioned in the issue itself. Should I remove it?

Yes, it's already in the repo topics so not necessary in the readme as this is not just a Hacktoberfest repo but a repo that will be maintained beyond Hacktoberfest

**Comment by arunkarthicknm** [comment]
Now?

**Pull request merged**